src/rfbzero/experiment.py

```diff
@@ -524,13 +524,19 @@ def _validate_protocol(
         if cell_model.init_ocv > 0.0 > self.voltage_limit_discharge:
             raise ValueError("Ensure that 'voltage_limit_discharge' >= 0.0 when 'init_ocv' > 0.0")
 
+        if crossover and cell_model.init_ocv > 0.0:
+            raise ValueError("Cannot use crossover mechanism for a full cell ('init_ocv' > 0.0)")
+
         if degradation is not None and (cls_degradation is not None or ncls_degradation is not None):
             raise ValueError("Cannot specify both 'degradation' and '(n)cls_degradation'")
 
         if degradation is not None:
             cls_degradation = degradation
             ncls_degradation = degradation
 
+        cls_degradation = copy.deepcopy(cls_degradation)
```

Comment:
looks good, but do we want a small comment here explaining why we use a deepcopy?